README.md

```diff
@@ -6,20 +6,24 @@ This is the officially supported .NET library for using Adyen's APIs.
-| [Adyen BinLookup API](https://docs.adyen.com/api-explorer/#/BinLookup/v52/overview)         | Endpoints for retrieving information, such as cost estimates, and 3D Secure supported version based on a given BIN. Current supported version |Adyen BinLookup API| **v52**|
-| [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/originKeys)    | This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version|Utility API| **v67**|
-| [Hosted Onboarding API](https://docs.adyen.com/api-explorer/#/Hop/v1/overview)              | Current supported version | Hosted Onboarding API | **v1** |
-| [Account API](https://docs.adyen.com/api-explorer/#/Account/v5/overview)                    | Current supported version | Account API | **v5** |
-| [Fund API](https://docs.adyen.com/api-explorer/#/Fund/v5/overview)                          | Current supported version|   Fund API  | **v5** |
-| [Terminal API (Cloud communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/cloud) | Our point-of-sale integration.| Cloud-based Terminal API | Cloud-based Terminal API       |      |
-| [Terminal API (Local communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/local) | Our point-of-sale integration.| Local-based Terminal API |   Local-based Terminal API     |      |
-| [POS Terminal Management API](https://docs.adyen.com/api-explorer/#/postfmapi/v1/overview)  | Endpoints for managing your point-of-sale payment terminals| POS Terminal Management API | **v1** |
+| API                                                                                                        | Description                                                                                                                                                                                                                                                       | Service Name                | Supported version        |
+|------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|--------------------------|
+| [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v70/overview)                         | Adyen Checkout API provides a simple and flexible way to initiate and authorise online payments. You can use the same integration for payments made with cards (including 3D Secure), mobile wallets, and local payment methods (for example, iDEAL and Sofort).  | Checkout API                | **v70**                  |
```

Comment:
Service names are not correct. They should match the services to include in your integration, not a copy paste of the first column